.travis.yml

```diff
@@ -12,6 +12,7 @@ language: node_js
 os:
   - linux
   - osx
+  - windows
```

Comment:
Regarding node versions below: do we use this library in production code aka actions? If not, we can drop 10.15 and 10.19.

Reply:
(I think it's used in prod).

Reply:
@alexkli It's in asset-compute-devtool

Reply:
Yes, so that's NOT an action.